src/main/java/net/dv8tion/jda/api/audit/ActionType.java

```diff
@@ -496,6 +494,38 @@
      */
     STAGE_INSTANCE_DELETE(85, TargetType.STAGE_INSTANCE),
 
+    /**
+     * An Administrator created an {@link net.dv8tion.jda.api.entities.sticker.Sticker Custom Sticker}.
```

Comment:
This should probably link to [GuildSticker](https://ci.dv8tion.net/job/JDA5/javadoc/net/dv8tion/jda/api/entities/sticker/GuildSticker.html) instead, and it's called guild sticker not custom sticker.